### PR TITLE
Added i2c-gpio example

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Comment by caternuson** [comment]
Looks generally OK. Did you actually test?

**Comment by dherrada** [comment]
@caternuson Yep, I did.

File: examples/bno055_i2c-gpio_simpletest.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+and does require the i2c-gpio kernel module to be
+installed and enabled. Most Raspberry Pis will
+already have it installed.
+"""
+
+import time
+from adafruit_extended_bus import ExtendedI2C as I2C
+import adafruit_bno055
+
+# Create library object using our Extended Bus I2C port
```
Comment (caternuson) [inline examples/bno055_i2c-gpio_simpletest.py:15]: Add comment here showing the line you used in config.txt to enable and configure the i2c-gpio overlay to go along with this example.

File: examples/bno055_i2c-gpio_simpletest.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"""
+This example demonstrates how to instantiate the
+Adafruit BNO055 Sensor using this library and just
+the I2C bus number.
+This example will only work on a Raspberry Pi
+and does require the i2c-gpio kernel module to be
+installed and enabled. Most Raspberry Pis will
+already have it installed.
```
Comment (caternuson) [inline examples/bno055_i2c-gpio_simpletest.py:8]: Add additional commentary here mentioning need to alter config.txt to actually enable the overlay:
```
To enable, modify config.txt...add `dtoverlay=i2c_gpio` etc...
```

**Comment by dherrada** [comment]
@caternuson This one https://www.raspberrypi.org/documentation/configuration/config-txt/ seems pretty good

**Comment by caternuson** [comment]
Thanks. This looks fine. If that page had more info specific to i2c-gpio, like how to set the pins, bus number, etc. then I'd say let's add the URL in a code comment. But, meh, it's pretty generic, so nevermind.

**Pull request merged**